### PR TITLE
Main Structure

### PR DESCRIPTION
**Pull request by ahmed-alfarouq**
What I did:
- Init webpack structure
- Add Style
- Add linters
- Add professional README file

### CODE REVIEW

**Review by helman101 — COMMENTED** [review]
# STATUS: INVALID 🔴⚠️ 

Hi @ahmed-al-farouq  😺 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not check for linter errors, your pull request should run a check for the following linters:
    image
instead i only see this:
    imagen
Kindly check your linters set-up and send another review request when your linters look like those on the first image. 🚀 



Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

gif

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: package.json [hunk 1]
```diff
@@ -0,0 +1,783 @@
...
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.56.1",
+    "webpack-cli": "^4.8.0",
+    "webpack-dev-server": "^4.3.1"
+  },
+  "scripts": {
+    "test": "jest",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
```
Comment (vichuge) [inline package.json]: - [ ] Everything looks great, but I'm afraid I couldn't install the project locally...
- I clone the repo and select the structure branch
- I delete the `package-lock.json`
- I ran `npm i` and I have the next error:
image
- For this same reason I couldn't use `npm start` (and this is a requirement for this project), please fix this bug. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "leader-board",
+  "version": "1.0.0",
+  "description": "This is an awesome Leader Board",
+  "main": "webpack.config.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "deploy": "gh-pages -d dist",
+    "build": "webpack"
+  },
```
Comment (vichuge) [inline package.json]: - [ ] Please, to run the project locally, add the instruction to run `npm start` or `npm run start`, in this way I could be able to run the project locally on my computer. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ♻️ ⚠️ ⚠️ 🚦 🚦 

Hi dear student, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

star_wars_almost_there


## Aspects to improve ♻️
_Check the comments under the review._

- [ ] Check the comments under the review. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lameck1 — APPROVED** [review]
Hi, @ahmed-al-farouq,

Your project is complete!  It's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**